Source/Atomic/UI/UI.h

```diff
@@ -100,6 +100,9 @@ class UI : public Object, private tb::TBWidgetListener
     void ShowDebugHud(bool value);
     void ToggleDebugHud();
 
+    // Cycle debug HUD between showing primitive stats, current mode, profiler data, all three or none
+    void CycleDebugHudMode();
+
```

Comment:
Yeah, I think UI.h is one of the more guilty files for method commenting problems :)  

I have drafting a simple easy to follow coding standard (which borrows heavily from the legacy one and hits the major stuff) on my shortlist